content/updates/2021-01/1-Accelarate-yourconnections-with-the-events-api.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_alt_img: "Events activation in the Connections"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Accelerate your connections with the Events API
+::: meta-data type="New" features="Connectivity" available="mid-January" link-to-doc="https://api.akeneo.com/events-documentation/introduction.html"
+
+You may know how crucial it is to efficiently connect your PIM to third-party apps such as your ERP, e-commerce platform, or DAM. To help you accelerate on this topic, we're releasing a brand new API, called the `Events API`.
+
+Coupled with our [existing REST API](https://api.akeneo.com/documentation/introduction.html), the Events API offers an additional and complementary way to synchronize PIM data with third-party applications. By reducing synchronization delays, it gives you the power to improve the time to market of your product information.
```

Comment:
```suggestion
Coupled with our [existing REST API](https://api.akeneo.com/documentation/introduction.html), the Events API offers an additional and complementary way to synchronize PIM data with third-party applications. By reducing synchronization delays, it gives you the power to accelerate the access and distribution of your product information.
```
rephrased the last sentence because "time to market of your product information" sounds weird, TTM can apply to products not to product information.

Reply:
In this case, I would prefer to go for "improve the time to market of your dear products." The idea is to link this technical feature to a real business goal.

Reply:
Like this:
```suggestion
Coupled with our [existing REST API](https://api.akeneo.com/documentation/introduction.html), the Events API offers an additional and complementary way to synchronize PIM data with third-party applications. By reducing synchronization delays, it gives you the power to improve the time to market of your dear products.
```

Reply:
Fixed with Laure's suggestion.